Description: Make sure cmake project name matches package name.
 This make sure the documentation end up in the correct
 directory.
Author: Petter Reinholdtsen <pere@debian.org>
Bug: https://github.com/ptrkrysik/gr-gsm/issues/304
Forwarded: https://github.com/ptrkrysik/gr-gsm/issues/304
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2017-08-09

--- gr-gsm-0.0.20170807.6c4304d.orig/CMakeLists.txt
+++ gr-gsm-0.0.20170807.6c4304d/CMakeLists.txt
@@ -22,7 +22,7 @@
 # Project setup
 ########################################################################
 cmake_minimum_required(VERSION 2.6)
-project(gr-grgsm CXX C)
+project(gr-gsm CXX C)
 enable_testing()
 
 #set(CMAKE_BUILD_TYPE "Debug")
